### ISSUE TITLE
Ractive.render does not render partials

### ISSUE THREAD

**Issue by berislavlopac** [ractivejs/ractive#1906]
With this template:

```mustache
{{#item}}
<form>

  <h3>{{full_name}}</h3>

  {{#fields}}
  <p>
    <label for="field_{{name}}">{{label}}:</label>
    {{> input_text}}
  </p>
  {{/fields}}

</form>
{{/item}}

{{#partial input_text}}
  <input type="text" name="{{name}}" id="field_{{name}}" value="{{item[name]}}">
{{/partial}}
```
if I call `teardown` and then `render` the partial doesn't get rendered. On the other hand, it renders correctly if the partial's code is embedded in the main template, like this:

```mustache
{{#item}}
<form>

  <h3>{{full_name}}</h3>

  {{#fields}}
  <p>
    <label for="field_{{name}}">{{label}}:</label>
    <input type="text" name="{{name}}" id="field_{{name}}" value="{{item[name]}}">
  </p>
  {{/fields}}

</form>
{{/item}}
```

Am I doing something wrong or is this a bug? Thanks!

**Comment by Madgvox** [comment]
On first impression this strikes me as very similar to https://github.com/ractivejs/ractive/issues/1904. I think there's something weird going on with rebuilding ractives.

**Comment by berislavlopac** [comment]
Possible, as `merge` solves the issue just the way it was suggested in #1904.

**Comment by Madgvox** [comment]
This is fixed as of 0.8-edge.
